src/api/ledger/parse/trustline.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const assert = require('assert');
+const utils = require('./utils');
+const flags = utils.core.Transaction.flags.TrustSet;
+
+function parseTrustline(tx) {
```

Comment:
Let's call it "ripple api" because it's unrelated to rest. There are five transaction types in the ripple api: `payment`, `trustline`, `order`, `orderCancellation`, and `settings`.